src/main/java/uk/gov/ons/ctp/common/rest/RestClient.java

```diff
@@ -295,7 +336,7 @@ public RestTemplate getRestTemplate() {
       MultiValueMap<String, String> queryParams,
       Object... pathParams)
       throws RestClientException {
```

Comment:
None of the methods now throw RestClientException, so in every case replace RestClientException with ResponseStatusException in the method signature and javadoc (or since they are spring exceptions which are unchecked you could just leave them off the signature/javadoc completely).

Reply:
Changed